src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -574,7 +628,8 @@ namespace winrt::TerminalApp::implementation
         if (isFirstTab)
```

Comment:
random q: shouldn't the tabview handle this now?

Reply:
Oh yea we don't need this anymore